style.css

```diff
@@ -1,7 +1,37 @@
+
 .header{
 	font-size: 36px;
 	letter-spacing: 1.2px;
 	font-family: helvetica;
 	color:#536240;
 	font-weight: 700;
+}
+.info{
+	position: relative;
+	display: flex;
+	left: 800px;
```

Comment:
What happens when you have a screen which doesn't have 800px? Rather than using left of 800px, try to use rigth of something. But It's not recommended to you left and right. try to fix this with something else. For example, if you have a header ( top of the page ), you have a body ( main content)  the page content. So you can have a header-container, and a main-container as a parent div. You can add everything that is related with the header into a div. So the header-container should have a logo, an h1, and a div with the info :) If you use display flex, with flex-direction: row, it will allign all the items in a row, and you won't need the left 800px anymore! As a best practice, try to rewrite the styleing than trying to repair it